internal/printer/printer.go

```diff
@@ -30,6 +31,8 @@ import (
 type Printer interface {
 	PkgPrintf(pkgPath types.UniquePath, format string, args ...interface{})
 	Printf(format string, args ...interface{})
+	IndentPrintf(indentation int, format string, a ...interface{})
+	IndentPrintfE(indentation int, format string, a ...interface{})
```

Comment:
One half-baked thought to avoid `XXXPrintf(` pattern here will be:

```
// introduce printer.Options struct with a convenient builder

type Options struct {
   Ident *bool
   PkgPath *types.UniquePath
   Err error
}

// Have a convenient builder for each of the above functions, for ex.
func (opts *Options)WithPkg(path types.UniquePath) *opts {
}

// Have generic printer API with the options

Printf(opts *Options, format string, a ...interface{})


```

Reply:
Updated the printer implementation